build.gradle.kts

```diff
@@ -10,13 +10,79 @@ plugins {
     kotlin("jvm") version Versions.kotlin apply false
     id("com.github.ben-manes.versions") version "0.36.0"
     id("com.cdsap.talaiot.plugin.base") version "1.4.1"
+    id("org.liquibase.gradle") version Versions.liquibaseGradlePlugin
+}
+
+val profile = properties.getOrDefault("profile", "dev")
+
+val props = java.util.Properties()
+val file = file("save-backend/src/main/resources/application-$profile.properties").apply { props.load(inputStream()) }
+
+if (File("secrets").exists()) {
+    file("secrets").apply { props.load(inputStream()) }
+}
+
+val databaseUrl = props.getProperty("spring.datasource.url")
+var username: String
+var password: String
+
+if (profile == "prod") {
+    username = props.getProperty("username")
+    password = props.getProperty("password")
+
+    fillSpringDatasource(username, password)
+} else {
+    username = props.getProperty("spring.datasource.username")
+    password = props.getProperty("spring.datasource.password")
+}
+
+fun fillSpringDatasource(userName: String, password: String) {
```

Comment:
Actually, I believe we don't need it - there is no point in filling sensitive data in properties file before build, while we already are planning to use docker secrets for spring services. The only thing that needs passwords during build is liquibase.